### PR TITLE
EWL-4557: Fix icons filling the screen by setting max widths and default widths…

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-4557: SG2 | Set max-width on SVGs](https://issues.ama-assn.org/browse/EWL-4557)

## Description
Fix icons filling the screen especially when we are doing backstop tests. 

## To Test
- [ ] `gulp serve`
- [ ] scroll down the page
- [ ] observe the icons are no longer filling the screen

## Visual Regressions
when you run `gulp backstop` there should be no new regressions introduced. 

## Relevant Screenshots/GIFs
screen shot 2018-02-20 at 11 21 55 am

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by chelsiejohnston** [comment]
@designerneil per our Standards doc, please update this PR to align to our standards regarding the following items before this PR can be reviewed:

### "Visual Regressions" section: 

Please make sure you are answering all the questions required by the template. You wrote:

> run gulp backstop

Simply saying "run gulp backstop" isn't helpful because it's a given (it's part of our understood team workflow for PR review), and also because it doesn't answer either of the questions the template requires, which are:

> 1. Does your work introduce any visual regressions to existing work in the Style Guide, or in a Drupal 8 environment? Please either call these out here or address them before marking your PR as ready for review.

An acceptable answer here might be "My work does not introduce visual regressions as far as I can tell after running `backstop test`" or "My work introduces a visual regression (explain it here) but this is a known issue and will be addressed in ticket xyz" or similar. 

> 2. If you are creating or updating a pattern be sure you have created or updated the scenario in backstop.json and have included a new or updated screenshot in bitmaps_reference.

Please point out the files/paths to updated tests that are included by this PR or otherwise explain why the work in the PR does not cover tests (e.g. "the scenarios didn't need to be updated because they were already there" or "this can't be tested visually").

Lastly, please use `code notation` when documenting command line commands.

Let me know when you've corrected the above and I'll re-mark this as `ready for review`.

**Comment by designerneil** [comment]
@chelsiejohnston Adding more verbose statement to whether or not I'm introducing visual regressions is not necessary here because the reviewer should always run gulp test or gulp backstop and see for themselves. No visual changes or tests were added in this PR. 

> Simply saying "run gulp backstop" isn't helpful because it's a given (it's part of our understood team workflow for PR review), and also because it doesn't answer either of the questions the template 

We should never introduce visual regressions. We are to review visual regressions ahead of time and update backstop ref images accordingly.

**Comment by designerneil** [comment]
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/116

**Comment by chelsiejohnston** [comment]
@designerneil ~if you want to make changes to our standards/PR process and argue your case there, feel free to bring this up in the FED meeting. For the time being though, this PR does not meet standards and should be edited accordingly. Thanks! ~

[EDIT] Sorry I did not see that you'd edited the Visual Regression section. This is fine now, carry on

**Comment by chelsiejohnston** [comment]
But I did not mean to close it just now, sorry about that!

File: styleguide/source/assets/icons/svg/icon-video.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 234 234" width="40" height="50"><path d="M202.4 48.5L157.8.4H42.6c-6 0-10.9 4.9-10.9 10.9v211.3c0 6 4.9 10.9 10.9 10.9h148.9c6 0 10.9-4.9 10.9-10.9V48.5zm-45.6-36.1L191 49.3h-28.9c-2.9 0-5.3-2.4-5.3-5.3V12.4zm35.4 207c0 2.9-2.4 5.3-5.3 5.3H47.1c-2.9 0-5.3-2.4-5.3-5.3V14.6c0-2.9 2.4-5.3 5.3-5.3h99.6v38c0 6 4.9 10.9 10.9 10.9h34.6v161.2z" fill="#faa634"/><path d="M142.9 107.7c0-10.8-8.8-19.6-19.6-19.6H75.6c-10.8 0-19.6 8.8-19.6 19.6v47.8c0 10.8 8.8 19.6 19.6 19.6h47.7c10.8 0 19.6-8.8 19.6-19.6v-11.6l27 27.3c.8.9 2 1.4 3.4 1.4 2.5 0 4.5-2 4.7-4.4v-.2-.1-72.5-.1c0-2.6-2.1-4.7-4.7-4.7-1.5 0-2.9.7-3.7 1.9L143 118.9v-11.2z" fill="#46166b"/></svg>
```
Comment (nstriedinger) [inline styleguide/source/assets/icons/svg/icon-video.svg:1]: @designerneil -- Should we remove all the inline width/heights so we can control with CSS?

**Pull request merged**